packages/frontend-shared/src/gql-components/composables/useCohorts.ts

```diff
@@ -64,15 +64,21 @@ export const useCohorts = () => {
   const getCohort = (config: CohortConfig) => {
     const cohortOptionSelected = ref<CohortOption>()
 
-    const cohortIds = config.options.map((option) => option.cohort)
+    const { name, options } = config
 
-    const fetchCohort = async () => {
-      const cohortSelected = await determineCohort(config.name, cohortIds)
+    if (!options || options.length <= 1) {
+      cohortOptionSelected.value = options?.[0]
+    } else {
+      const cohortIds = options.map((option) => option.cohort)
 
-      cohortOptionSelected.value = config.options.find((option) => option.cohort === cohortSelected.data?.determineCohort?.cohort)
-    }
+      const fetchCohort = async () => {
+        const cohortSelected = await determineCohort(name, cohortIds)
```

Comment:
Do we still need to determine the cohort? Isn't there only 1 cohort now that the A/B testing ended? Is this now dead code or a no-op?

Reply:
I think we should add a comment that the infrastructure is left is place as future a/b testing is expected within this same place. I recall asking this somewhere in the original issue.

Reply:
Correct, since there are no active experiments this is currently dead code. However, product wants the ability to quickly introduce new experiments so it was requested that the logic be maintained. I will add some comments to this area to clarify